### ISSUE TITLE
Blue background color and select color are too similar

### ISSUE THREAD

**Issue by felixbinder** [zoxon/vscode-theme-monokai-light#4]
VS Code 1.57.1, macOS Big Sur

The text of partial search matches in the top menu is the same as the highlight color, making this hard to parse.

![Screen Shot 2021-06-25 at 14 23 23](https://user-images.githubusercontent.com/24477285/123424429-1dc84280-d5c1-11eb-82d3-a71ee5c7bef1.png)

**Comment by felixbinder** [comment]
Another example:

![Screen Shot 2021-06-25 at 15 20 17](https://user-images.githubusercontent.com/24477285/123430833-ea89b180-d5c8-11eb-9ff3-82959b0d381c.png)

**Comment by zoxon** [comment]
Thank you. 
I will fix this but if you want to help... 
PR's are welcome
